internal/context/user_plane_information.go

```diff
@@ -421,6 +424,43 @@ func (upi *UserPlaneInformation) LinksFromConfiguration(upTopology *factory.User
 	}
 }
 
+func (upi *UserPlaneInformation) UpNodeDelete(upNodeName string) {
+	upNode, ok := upi.UPNodes[upNodeName]
+	if ok {
+		logger.InitLog.Infof("UPNode [%s] found. Deleting it.\n", upNodeName)
+		if upNode.Type == UPNODE_UPF {
+			logger.InitLog.Tracef("Delete UPF [%s] from its NodeID.\n", upNodeName)
+			RemoveUPFNodeByNodeID(upNode.UPF.NodeID)
+			if _, ok = upi.UPFs[upNodeName]; ok {
+				logger.InitLog.Tracef("Delete UPF [%s] from upi.UPFs.\n", upNodeName)
+				delete(upi.UPFs, upNodeName)
+			}
+			for selectionStr, destMap := range upi.DefaultUserPlanePathToUPF {
+				for destIp, path := range destMap {
+					if nodeInPath(upNode, path) != -1 {
+						logger.InitLog.Infof("Invalidate cache entry: DefaultUserPlanePathToUPF[%s][%s].\n", selectionStr, destIp)
+						delete(upi.DefaultUserPlanePathToUPF[selectionStr], destIp)
+					}
+				}
+			}
+		}
+		if upNode.Type == UPNODE_AN {
+			logger.InitLog.Tracef("Delete AN [%s] from upi.AccessNetwork.\n", upNodeName)
+			delete(upi.AccessNetwork, upNodeName)
+		}
+		logger.InitLog.Tracef("Delete UPNode [%s] from upi.UPNodes.\n", upNodeName)
+		delete(upi.UPNodes, upNodeName)
+
+		// update links
+		for name, n := range upi.UPNodes {
+			if index := nodeInLink (upNode, n.Links); index != -1 {
```

Comment:
Hi @tim-ywliu , can you please help to fix this lint issue? Thanks.

Reply:
```
gofmt -s -w <file>
```